packages/cli/bin/run

```diff
@@ -2,6 +2,5 @@
 
 process.env.HEROKU_UPDATE_INSTRUCTIONS = process.env.HEROKU_UPDATE_INSTRUCTIONS || 'update with: "npm update -g heroku"'
 
-const oclif = require('@oclif/core')
-
-oclif.run().then(require('@oclif/core/flush')).catch(require('@oclif/core/handle'))
+require('@oclif/command').run()
+.then(require('@oclif/command/flush'))
```

Comment:
This is the critical part of this PR. It reverts to pre-core for running commands in the CLI. Everything works this way, and the only known trade-off is that it is no longer maintained. This is a short-term solution until v5 plugins can be upgraded.